docs/components/cache.rst

```diff
@@ -1,4 +1,107 @@
 Cache
 #####
 
+Symfony makes heavy use of a filesystem cache. When developing for Mautic, clearing the cache is a regular occurrence. By default, Mautic instances have the cache located in ``var/cache/ENV`` where ``ENV`` is the environment currently accessed (``dev`` or ``prod``). To rebuild the cache, delete the relevant ``ENV`` folder within the cache directory, or run the Symfony command ``php bin/console cache:clear --env=ENV``. If a specific environment isn't passed to the command via ``--env=ENV``, Mautic uses the ``dev`` environment by default.
+
+In the ``dev`` environment, Mautic doesn't cache translations, views, and Assets. However, changes to these files require clearing the cache for them to take effect in the ``prod`` environment. Changes to Mautic config files, Symfony config files, etc., require clearing of the cache regardless of the environment.
+
+The typical rule of thumb is, if Mautic isn't acting as you expect after making changes, try clearing your cache. If you get ``class could not be found`` or ``cannot redeclare class`` errors when using the ``cache:clear`` command, manually delete the ``var/cache/ENV`` folder then run the command and/or browse to the site to rebuild.
+
+Cache bundle
+************
+
+Enables PSR-6 and PSR-16 caching. Check :xref:`Symfony Cache Component`
```

Comment:
in preview mode in github, this is showing like `[:xref:`Symfony Cache Component`](https://github.com/fakela/developer-documentation-new/blob/cache/docs/components/cache.rst#id1)`

Reply:
We use Gitpod for previewsing because it allows us to render the entire docs.

Here's some documentation on how to use Gitpod (we should really write this up for the docs!): https://contribute.mautic.org/contributing-to-mautic/tester#the-easy-way-using-gitpod

When you open it up in Gitpod you'll see the button top right to preview:

![screenshot-fakela-developerdocumen-p4sgachbel9 ws-eu107 gitpod io-2024 01 22-13_50_56](https://github.com/mautic/developer-documentation-new/assets/2930593/5d52f9bc-4364-49b1-9f31-94ae83c55280)

It then looks like this:

![screenshot-fakela-developerdocumen-p4sgachbel9 ws-eu107 gitpod io-2024 01 22-13_51_56](https://github.com/mautic/developer-documentation-new/assets/2930593/0f7fbd8d-2911-440e-8ff3-dad18d923642)

It's pretty handy to check out the layout quickly, and also to test out what changes would look like! You can't use the nav on the left, or the search, in preview mode - you'd have to open the relevant file and preview it again.

Hope that helps! (you can also build locally if you prefer, the docs are in the readme for the repo).